playground/mocks/data/idp/idx/error-identify-with-piv.json

```diff
@@ -0,0 +1,66 @@
+{
+  "stateHandle": "eyJ6aXAiOiJERUYiLCJhbGlhcyI6ImVuY3J5cHRpb25",
+  "version": "1.0.0",
+  "expiresAt": "2020-04-13T22:55:53.000Z",
+  "intent": "LOGIN",
+  "messages": {
+    "type": "array",
+    "value": [
+      {
+        "message": "Authentication failed",
+        "class": "ERROR"
```

Comment:
Do we have a ticket tracking the missing `i18n` key here?

Reply:
Just fixed that! I don't have a working backend yet, so used a similar key for now